webrtc.html

```diff
@@ -399,23 +399,30 @@
 
         <ol>
           <li>
-            <p>Append an object representing the current call being handled
-            (i.e. function name and corresponding arguments) to the
-            <code>operations</code> array.</p>
+            <p>Let <var>p</var> be the result of transforming
+            <code>operations</code> by a fulfillment handler that runs the
+            following steps:</p>
+            <ol>
+              <li>
+                <p>If <var>connection</var>'s <a href=
+                "#dom-peerconnection-signaling-state"><code>RTCPeerConnection</code>
+                signalingState</a> is <code>closed</code>, abort these steps.</p>
+              </li>
+              <li>
+                <p>Return the result of executing the current call
+                being handled.</p>
+              </li>
+            </ol>
           </li>
 
           <li>
-            <p>If the length of the <code>operations</code> array is exactly 1,
-            execute the function from the front of the queue
-            asynchronously.</p>
+            <p>Set <var>connection</var>'s <code>operations</code> to the result
+            of transforming <var>p</var> by a rejection handler that returns
+            <var>undefined</var>.</p>
```

Comment:
Use english too please.  You are saying that you need to ensure that errors from the last operation in the chain need to be handled with `.catch()` so that subsequent operations aren't affected by earlier failures.